rust/src/db/keyspace.cql

```diff
@@ -0,0 +1 @@
+CREATE KEYSPACE IF NOT EXISTS carepet WITH replication = { 'class': 'NetworkTopologyStrategy', 'replication_factor': '3' } AND durable_writes = TRUE;
```

Comment:
nitpick: durable_writes defaults to true and it's not going to change ever (it's crucial for Scylla/Cassandra to be fault-tolerant), so I'm pretty sure you can safely drop it from the keyspace definition. It doesn't hurt though